### PR TITLE
Evan update notifications 20200323

### PR DESCRIPTION
**Pull request by bin315a1**
## Changes proposed in this pull request
1. Added notifications to everywhere necessary
2. Fixed all bugs and edge cases, converted to async/promise format in Request models, 
3. Added error response formalities
4. Test scripts for Noti and Request model

@bruinpool-devs

### CODE REVIEW

File: src/noti/controller.js [hunk 1]
```diff
@@ -1,45 +1,54 @@
...
+    try {
+      const res = await Noti.find({ username, viewed: false }).sort({date: -1});
+      return resolve(res);
+    } catch(err) {
+      return reject(err)
     }
   })
-    .sort({ _id: -1 })
-    .limit(8);
+  // Noti.find({ username, viewed: false }, (err, result) => {
```
Comment (bin315a1) [inline src/noti/controller.js:12]: erase

File: src/noti/controller.js [hunk 2]
```diff
@@ -1,45 +1,54 @@
...
-        callback(null, result);
-      }
+      return resolve(res);
+    } catch (err) {
+      return reject(err);
     }
-  )
-    .sort({ _id: -1 })
-    .limit(8);
+  });
```
Comment (bin315a1) [inline src/noti/controller.js:35]: sort and limit
